cluster_client_test.go

```diff
@@ -1,95 +0,0 @@
-package redis
-
-import (
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-)
-
-var _ = Describe("ClusterClient", func() {
```

Comment:
This test was actually quite useful to get the cluster basics right. I have no 'personal attachment' but keeping it in some way would probably improve my sleep :)